jobs/build-arch.Jenkinsfile

```diff
@@ -306,6 +306,15 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
             pipeutils.build_artifacts(pipecfg, params.STREAM, basearch)
         }
 
+        // secex specific tests
+        if (shwrapCapture("cosa meta --get-value images.qemu-secex") != "None") {
+            stage("Kola:TestSecex") {
+                // FIXME: Once we have valid HKD for the builder we should add `--qemu-secex-hostkey /path/to/key` switch and enable reboot tests
+                kola(cosaDir: env.WORKSPACE, arch: basearch, skipBasicScenarios: true, skipSecureBoot: true, skipUpgrade: true,
+                     extraArgs: "--qemu-secex --tag secex")
```

Comment:
I'm not that into the implementation of the kola function, but would it not make more sense to implement a setting here like `runOnlySecex: true`, to specify that we only want secex tests to run? This way, with all the skips seems more prone to error when future tests are added.

Reply:
maybe, but than i have to update https://github.com/coreos/coreos-ci-lib/blob/main/vars/kola.groovy

Reply:
@jlebon wdyt?

Reply:
Probably not worth having a helper knob specific to secex. I could imagine in the future though have a `tag` parameter to make this more ergonomic. Anyway, in my review below I mention we can simplify this line today.